.env.example

```diff
@@ -0,0 +1,29 @@
+# Example for Sepolia configuration.
+
+# The rollup RPC.
+ROLLUP_RPC=http://0.0.0.0:5050
+
+# For sovereign mode, use STARKNET_RPC instead:
+STARKNET_RPC=http://0.0.0.0:5050
+
+# Celestia node.
+CELESTIA_RPC=http://0.0.0.0:26658
+CELESTIA_TOKEN=
+
+# Herodotus Dev (atlantic + integrity cairo verifier).
+# Create an account here for API key: https://staging.dashboard.herodotus.dev.
+#
+# Reference for deployed contracts:
+# https://github.com/HerodotusDev/integrity/blob/95fa6cd82b28086f4676070ced5fcb10db4b73a5/deployed_contracts.md
+# Currently, the layout bridge is verified using `recursive_with_poseidon` verifier.
+ATLANTIC_KEY=
+SETTLEMENT_INTEGRITY_ADDRESS=0x05E529706944049BB2Be637a26A4d78b32e554Ecaa54D0e608F2Fa9f1472c516
+# Use the scripts in `./scripts` to compile the cairo programs.
+SNOS_PROGRAM=./programs/snos.json
+LAYOUT_BRIDGE_PROGRAM=./programs/layout_bridge.json
```

Comment:
At some point, we may have the default value pointing to them with `clap`. Or let's keep nothing as default?